server/apps/publicapi_publish/packages.py

```diff
@@ -12,7 +12,10 @@
 
 
 class PublicPackagesResource(PackagesResource):
-    datasource = {'source': 'items'}
+    datasource = {
+        'search_backend': 'elastic',
+        'source': 'items'
+    }
     item_methods = ['DELETE', 'PATCH', 'GET', 'DELETE']
```

Comment:
Why we have DELETE and PATCH? My impression was that public api will be readonly.
There are 2 DELETE in the item_methods.